iis-wf/iis-wf-primary/src/test/resources/eu/dnetlib/iis/wf/primary/main/output-validation/oozie_app/workflow.xml

```diff
@@ -0,0 +1,90 @@
+<workflow-app xmlns="uri:oozie:workflow:0.4" name="test-primary_main_output_validation">
+
+    <global>
+        <job-tracker>${jobTracker}</job-tracker>
+        <name-node>${nameNode}</name-node>
+        <configuration>
+            <property>
+                <name>mapreduce.job.queuename</name>
+                <value>${queueName}</value>
+            </property>
+        </configuration>
+    </global>
+
+    <start to="primary_main"/>
+
+    <action name="primary_main">
+        <sub-workflow>
+            <app-path>${wf:appPath()}/primary_main</app-path>
+            <propagate-configuration/>
+            <configuration>
+                <property>
+                    <name>import_islookup_service_location</name>
+                    <value>$UNDEFINED$</value>
+                </property>
+                <property>
+                    <name>import_infospace_graph_location</name>
+                    <value>$UNDEFINED$</value>
+                </property>
+                <property>
+                    <name>import_patent_tsv</name>
+                    <value>$UNDEFINED$</value>
+                </property>
+                <property>
+                    <name>ingest_pmc_cache_location</name>
+                    <value>$UNDEFINED$</value>
+                </property>
+                <property>
+                    <name>metadataextraction_cache_location</name>
+                    <value>$UNDEFINED$</value>
+                </property>
+                <property>
+                    <name>software_webcrawl_cache_location</name>
+                    <value>$UNDEFINED$</value>
+                </property>
+                <property>
+                    <name>patent_cache_location</name>
+                    <value>$UNDEFINED$</value>
+                </property>
+                <property>
+                    <name>citationmatchingFuzzyNumberOfPartitions</name>
+                    <value>$UNDEFINED$</value>
+                </property>
+                <property>
+                    <name>export_patent_date_of_collection</name>
+                    <value>$UNDEFINED$</value>
+                </property>
+                <property>
+                    <name>output_remote_location</name>
+                    <value>/user/root/</value>
```

Comment:
This would work for sure but I am wondering whether we could somehow make sure we have a full control over testing environment.

What I mean is once the `/user/root/` directory is missing (due to whatever reason, e.g. running tests in a test cluster without this particular directory) our test might give unexpected results. 

I guess it will be better if we could create such directory for testing purposes and e.g. apply `chmod` of `<fs>` to restrict the privileges (e.g. by disallowing write access). We would need to drop this directory after the test execution to make sure the next iteration will not fail when cleaning up the env.

Reply:
Argh, you are right, I've been running tests on main cluster where this dir is present, but ci cluster does not have `root` user and therefore `/user/root` does not exist. I'm changing this into temporary dir created within init action with write permissions removed just as you commented.